.editorconfig

```diff
@@ -0,0 +1,5 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
```

Comment:
this allows consistent formatting, the plugin is available in many different editors, see https://editorconfig.org/